core-bundle/src/String/HtmlAttributes.php

```diff
@@ -132,10 +129,7 @@ public function addClass(string ...$classes): self
             ' ',
             array_unique(
                 array_filter(
-                    array_merge(
-                        explode(' ', $this->attributes['class'] ?? ''),
-                        array_map('trim', $classes)
-                    )
+                    explode(' ', ($this->attributes['class'] ?? '').' '.implode(' ', $classes)),
```

Comment:
```suggestion
                    preg_split('/[ \t\n\r\f]+/', ($this->attributes['class'] ?? '').' '.implode(' ', $classes)),
```

This would be all “supported” whitespace if we want to support this? 
https://drafts.csswg.org/selectors/#whitespace

_(all `explode()`s would have to be changed then I think)_

Reply:
If we're already here. Why not go the full way. :relaxed: 
See f8e25174e14819d03f017a0a1920a21b8a52ff02.